src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -47,8 +46,7 @@ public abstract class BaseCommand
         /// <summary>
         /// Gets or sets the GitHub token used to submit a pull request on behalf of the user.
         /// </summary>
-        [Option('t', "token", Required = false, HelpText = "GitHubToken_HelpText", ResourceType = typeof(Resources))]
-        public string GitHubToken { get; set; }
+        public virtual string GitHubToken { get; set; }
```

Comment:
Is there any point to having this on the base class at all anymore?

Reply:
I think you're right. Only the SetAndCheckGitHubToken method uses this property, which we can easily make into a parameter. Removed this virtual property from the base class

Reply:
Oops, that won't work, since that method actually sets this.GitHubToken internally. If you just change it to a parameter it won't change the value

Reply:
```
Launching web browser... If default web browser does not launch, please go to: [https://github.com/login/device](https://github.com/login/device)
Please enter the following user verification code when prompted: CAAD-3153

Resuming command execution...
No token provided, submission to GitHub skipped.
```

Reply:
Ahh, thanks for catching my mistake, I reverted it back to what we had before. I think we'll need to keep this around for now. I'll look into this further when I refactor some of the oauth stuff in a future PR.